### DISCUSSION TITLE
I can't open the project

### DISCUSSION THREAD

**Discussion by jurkax** [jomjol/AI-on-the-edge-device#3056] [General]
Please, give me an advice.

I can't open the project, Output from Visual studio code:

Resolving esp32cam dependencies...
Already up-to-date.
Updating metadata for the vscode IDE...
UserSideException: Processing esp32cam (board: esp32cam; framework: espidf; platform: platformio/espressif32 @ 6.5.0)
--------------------------------------------------------------------------------
Verbose mode can be enabled via `-v, --verbose` option
CONFIGURATION: https://docs.platformio.org/page/boards/espressif32/esp32cam.html
PLATFORM: Espressif 32 (6.5.0) > AI Thinker ESP32-CAM
HARDWARE: ESP32 240MHz, 320KB RAM, 4MB Flash
DEBUG: Current (cmsis-dap) External (cmsis-dap, esp-bridge, esp-prog, iot-bus-jtag, jlink, minimodule, olimex-arm-usb-ocd, olimex-arm-usb-ocd-h, olimex-arm-usb-tiny-h, olimex-jtag-tiny, tumpa)
PACKAGES: 
 - framework-espidf @ 3.50102.240122 (5.1.2) 
 - tool-cmake @ 3.16.4 
 - tool-esptoolpy @ 1.40501.0 (4.5.1) 
 - tool-idf @ 1.0.1 
 - tool-mconf @ 1.4060000.20190628 (406.0.0) 
 - tool-ninja @ 1.9.0 
 - tool-riscv32-esp-elf-gdb @ 11.2.0+20220823 
 - tool-xtensa-esp-elf-gdb @ 11.2.0+20230208 
 - toolchain-esp32ulp @ 1.23500.220830 (2.35.0) 
 - toolchain-xtensa-esp32 @ 12.2.0+20230208
Reading CMake configuration...
-- Found Git: C:/Programy/Git/cmd/git.exe (found version "2.44.0.windows.1") 
-- git rev-parse returned 'fatal: not a git repository (or any of the parent directories): .git'
-- Component directory C:/Cajka/dokumenty/data_esp/AI-on-the-edge-device/code/components/esp-nn does not contain a CMakeLists.txt file. No component will be added
-- Component directory C:/Cajka/dokumenty/data_esp/AI-on-the-edge-device/code/components/esp-tflite-micro does not contain a CMakeLists.txt file. No component will be added
-- Component directory C:/Cajka/dokumenty/data_esp/AI-on-the-edge-device/code/components/esp32-camera does not contain a CMakeLists.txt file. No component will be added
-- Component directory C:/Cajka/dokumenty/data_esp/AI-on-the-edge-device/code/components/stb does not contain a CMakeLists.txt file. No component will be added
-- The C compiler identification is GNU 12.2.0
-- The CXX compiler identification is GNU 12.2.0
-- The ASM compiler identification is GNU
-- Found assembler: C:/Users/jiri.cajka/.platformio/packages/toolchain-xtensa-esp32/bin/xtensa-esp32-elf-gcc.exe
-- Check for working C compiler: C:/Users/jiri.cajka/.platformio/packages/toolchain-xtensa-esp32/bin/xtensa-esp32-elf-gcc.exe
-- Check for working C compiler: C:/Users/jiri.cajka/.platformio/packages/toolchain-xtensa-esp32/bin/xtensa-esp32-elf-gcc.exe -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: C:/Users/jiri.cajka/.platformio/packages/toolchain-xtensa-esp32/bin/xtensa-esp32-elf-g++.exe
-- Check for working CXX compiler: C:/Users/jiri.cajka/.platformio/packages/toolchain-xtensa-esp32/bin/xtensa-esp32-elf-g++.exe -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Building ESP-IDF components for target esp32
-- Configuring incomplete, errors occurred!
See also "C:/Cajka/dokumenty/data_esp/AI-on-the-edge-device/code/.pio/build/esp32cam/CMakeFiles/CMakeOutput.log".

fatal: not a git repository (or any of the parent directories): .git
CMake Error at C:/Users/jiri.cajka/.platformio/packages/framework-espidf/tools/cmake/build.cmake:266 (message):
  Failed to resolve component 'esp-tflite-micro'.
Call Stack (most recent call first):
  C:/Users/jiri.cajka/.platformio/packages/framework-espidf/tools/cmake/build.cmake:302 (__build_resolve_and_add_req)
  C:/Users/jiri.cajka/.platformio/packages/framework-espidf/tools/cmake/build.cmake:303 (__build_expand_requirements)
  C:/Users/jiri.cajka/.platformio/packages/framework-espidf/tools/cmake/build.cmake:303 (__build_expand_requirements)
  C:/Users/jiri.cajka/.platformio/packages/framework-espidf/tools/cmake/build.cmake:595 (__build_expand_requirements)
  C:/Users/jiri.cajka/.platformio/packages/framework-espidf/tools/cmake/project.cmake:547 (idf_build_process)
  CMakeLists.txt:22 (project)

========================= [FAILED] Took 18.41 seconds =========================

Environment    Status    Duration
-------------  --------  ------------
esp32cam       FAILED    00:00:18.406
==================== 1 failed, 0 succeeded in 00:00:18.406 ====================

**Comment by SybexX** [comment]
https://github.com/jomjol/AI-on-the-edge-device/discussions/3039#discussioncomment-9215857
